src/agent/onefuzz/src/expand.rs

```diff
@@ -93,6 +93,10 @@ impl PlaceHolder {
     }
 }
 
+pub trait GetExpand {
+    fn get_expand(&self) -> Result<Expand<'_>>;
+}
```

Comment:
I would say don't add the trait, since it is never used to abstract over implementations (we never use `dyn GetExpand` or `T: GetExpand` or `impl GetExpand` anywhere). Just add the `get_expand` function to the `impl T`.

Reply:
I think this is the right move, I don't envision any cases where it could be useful for an abstraction.